src/renderer/components/Onboarding/Screens/Tutorial/screens/QuizSuccess.js

```diff
@@ -71,7 +71,7 @@ export function QuizSuccess({ sendEvent, context }: Props) {
             {t("onboarding.screens.tutorial.screens.quizSuccess.buttons.prev")}
           </Text>
         </Button>
-        <Button id="quizz-success-cta" inverted primary onClick={() => sendEvent("NEXT")}>
+        <Button data-test-id="quizz-success-cta" inverted primary onClick={() => sendEvent("NEXT")}>
```

Comment:
```suggestion
        <Button data-test-id="quiz-success-cta" inverted primary onClick={() => sendEvent("NEXT")}>
```

Reply:
Forget it, I didn't notice the original file is misspelled too. Not worth the hassle of changing